src/VerifyU2FController.js

```diff
@@ -69,25 +79,62 @@ function (Okta, FormController, FormType, FooterSignout, Q) {
       className: 'verify-u2f-form',
       noCancelButton: true,
       save: _.partial(Okta.loc, 'verify.u2f.retry', 'login'),
+      noButtonBar: function () {
+        return !window.hasOwnProperty('u2f');
+      },
       modelEvents: {
         'error': '_showRetry',
         'request': '_hideRetry'
       },
 
-      formChildren: [
-        FormType.View({
-          View: '\
+      formChildren: function () {
```

Comment:
I created https://oktainc.atlassian.net/browse/OKTA-106499 to follow up on it. I think we can refactor some part of it. 
When we started I didn't realize they will share enough logic, so the design since the beginning was to have them independent. My bad.